test/parallel/test-cli-eval.js

```diff
@@ -66,6 +66,12 @@ child.exec(nodejs + ' --eval "require(\'./test/parallel/test-cli-eval.js\')"',
       assert.equal(status.code, 42);
     });
 
+// Missing argument should not crash
+child.exec(nodejs + ' -e', function(status, stdout, stderr) {
```

Comment:
As the assertion happens within the callback, its better to use `common.mustCall`